routes/experimenters.js

```diff
@@ -30,3 +31,39 @@ exports.create = function (req, res) {
       });
    });
 };
+
+function updateExperimenter(req, res, exp) {
+    Experimenter.update({ _id : ObjectId(req.params.id) }, exp, function (err, numUpdated, rawRes) {
+       if (err) {
+          log.error('Error on PUT /experimenters/' + req.params.id, err);
+          return res.send(500);
+       }
+
+       if(numUpdated == 0) {
+           // no experimenters updated, bad PUT request
+           return res.send(400);
+       }
+
+       return res.status(200).send(rawRes);
+    });
+}
+
+// PUT /experimenters/:id
+exports.update = function(req, res) {
+    var exp = {};
+    if(req.body.username !== undefined) exp.username = req.body.username;
+    if(req.body.name !== undefined) exp.name = req.body.name;
+    if(req.body.email !== undefined) exp.email = req.body.email;
```

Comment:
I prefer checking for truthy values on these: `if (req.body.username) exp.username = req.body.username;`